### PR TITLE
:broom: Update magetasks & dependencies, rm LICENSE files

### PR DESCRIPTION
**Pull request by cardil**
# Changes

- :broom: Update magetasks & dependencies
- :broom: Remove the `LICENSE` files
- :broom: Fix lint errors
- :broom: Adjust to the knative/client-pkg#155

/kind cleanup

Closes #333

### CODE REVIEW

File: build/go.mod [hunk 1]
```diff
@@ -3,138 +3,205 @@ module knative.dev/kn-plugin-event/build
...
+	knative.dev/pkg v0.0.0-20240327140624-1ca1f09c329e // indirect
+	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
+	sigs.k8s.io/kind v0.22.0 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
 replace knative.dev/kn-plugin-event => ../
+
+replace knative.dev/client-pkg => github.com/cardil/knative-client-pkg v0.0.0-20240328130510-cf596594c1b3
```
Comment (cardil) [inline build/go.mod:207]: Remove after https://github.com/knative/client-pkg/issues/173 is fixed

/hold

Comment (cardil) [reply build/go.mod]: Fixed with https://github.com/knative-extensions/kn-plugin-event/compare/067f03411eb06bbcadc754e0eb6ac2d8f4c0c0a0..565965f5f1f84651e2c67d13bd8c7c67f572988c

**Review by dsimansk — APPROVED** [review]

File: build/go.mod (hunk 1 above)
Comment (cardil) [reply build/go.mod]: /unhold

**Comment by cardil** [comment]
/override codecov/patch

That Codecov is weird!

**Pull request merged**